### PR TITLE
homepage

### PR DESCRIPTION
**Pull request by RoseBatoeva**

### CODE REVIEW

File: styles/homepage.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+.map {
+    padding-left: 20px;  /*расстояние между текстом и краями страницы в телефоне */
+    padding-right: 20px; /*расстояние между текстом и краями страницы в телефоне */
+    display: flex;
+    margin-bottom: -2rem;
+} 
+
+@media (min-width: 576px){
+    .homepage {
```
Comment (trc485) [inline styles/homepage.css:21]: cancel this class homepage and the code here.
We just need a container for the main part of any page.
So this rule should be in index.css
example
```css
.main {
        max-width: 960px;
        margin-left: auto;
        margin-right: auto;
}
```

File: index.html [hunk 2]
```diff
@@ -33,7 +35,49 @@
         </div>
       </div>  
     </div>
-
+<!-- ///////////////////////////////////////////////////////////////////////////// -->
+<div class="homepage">
```
Comment (trc485) [inline index.html:39]: change this to

<main class="main">

File: styles/homepage.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
+.homepage-section {
```
Comment (trc485) [inline styles/homepage.css:1]: You will have "sections" in every page.
Can't. you make this rule reusable?
create class .section
```css
.section {
   /* I think you need these paddings only for narrow screen so remove them in larger screen*/
    padding-left: 20px;
    padding-right: 20px;
    /* margin-bottom? */

}
```

File: styles/homepage.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    padding-left: 20px;  /*расстояние между текстом и краями страницы в телефоне */
+    padding-right: 20px; /*расстояние между текстом и краями страницы в телефоне */
+}
+.counter {
+    width: 100%;
+}
+
+.contacts {
```
Comment (trc485) [inline styles/homepage.css:9]: this is a duplication of homepage-section, so reuse same class

File: index.html [hunk 5]
```diff
@@ -33,7 +35,49 @@
...
+    <div class="homepage-section">
```
Comment (trc485) [inline index.html:40]: you can make use of html tag section
```html
<section class="section">
```
Here and in other places too

File: styles/homepage.css (hunk 1 above)
Comment (RoseBatoeva) [reply styles/homepage.css:21]: Where in the index.hrml should I put class="main" ?

Comment (trc485) [reply styles/homepage.css:21]: <main> is same a a <div>
In this page main is everything between nav and footer

File: styles/homepage.css (hunk 3 above)
Comment (RoseBatoeva) [reply styles/homepage.css:1]: I removed them no?
.homepage {
 margin-left: auto;
 margin-right: auto;
}

**Pull request merged**